src/pudl/transform/ferc1.py

```diff
@@ -2629,6 +2652,30 @@ def process_dbf(self, raw_dbf: pd.DataFrame) -> pd.DataFrame:
         return df
 
 
+class ElectricEnergyAccountDispositionsFerc1TableTransformer(
```

Comment:
Should we get rid of the `Account` from the class names too, to match the PUDL table names as we do elsewhere?

Reply:
ah yes! lol good catch